### PR TITLE
Added informative resources for blockchain

### PR DESCRIPTION
**Pull request by kashc11**

### CODE REVIEW

**Comment by 3scava1i3r** [comment]
Hi there could you please add links to great blogs from hackernoon and not just hackernoon.com, and add the issue number inside your PR with your discord handle.Join our [discord](https://discord.gg/hd7gvVr) as well.For further info take a look at [contributing guidlines.md](https://github.com/1gne0u5/Learning-Resources/blob/master/Contributing%20Guidelines.md)

**Comment by 3scava1i3r** [comment]
this is great work but add all of this in Blockchain/README.md

**Comment by kashc11** [comment]
> this is great work but add all of this in Blockchain/README.md

Sure!

File: Blockchain/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
 [![Maintenance](https://img.shields.io/maintenance/yes/2020?style=flat-square)]()
 
-
+Issue no: #37
```
Comment (3scava1i3r) [inline Blockchain/README.md:4]: do not write the issue number here

File: Blockchain/README.md [hunk 2]
```diff
@@ -14,8 +14,13 @@ Learning Resources-
 - Introducing Ethereum and Solidity: Foundations of Cryptocurrency and Blockchain Programming for Beginners
 - Investing in Ethereum: The Ultimate Guide to Learning–And Profiting From–Cryptocurrencies
 - Complete Guide to Understanding Ethereum, Blockchain, Smart Contracts, ICOs, and Decentralized Apps. Includes guides on buying Ether,
-Cryptocurrencies and Investing in ICOs
+  Cryptocurrencies and Investing in ICOs
```
Comment (3scava1i3r) [inline Blockchain/README.md:17]: try to see the preview and use 
- xyz for bullet points(use this before statement "- ")

**Comment by 3scava1i3r** [comment]
added some remarks 
i think there was some error when you were at the discord server please try to rejoin here https://discord.gg/6QAjyBa

**Pull request merged**